mygithub/src/components/github/Profile.js

```diff
@@ -1,58 +1,59 @@
-import React,{Component} from 'react';
+import React, { Component } from 'react';
 import RepoList from './RepoList.js';
-class Profile extends Component{
-	
-	render(){
-// Nesting profile component in App component
-		return(
+class Profile extends Component {
+
+	render() {
+		// Nesting profile component in App component
+		return (
 			<div>
 				<div className="panel panel-default">
-					  <div className="panel-heading"></div>
-					  <div className="panel-body">
-					    <h1> Name :{this.props.userData.name} </h1>
-					  </div>
+					<div className="panel-heading"></div>
+					<div className="panel-body">
+						<h1> Name :{this.props.userData.name} </h1>
+					</div>
```

Comment:
Minor suggestion: you could reduce some nesting by combining the two `h3` tags within the `panel-heading` div into a single line, like this:
```html
<div className="panel-heading">
  <h3 className="panel-title">Github Profile - Profile</h3>
</div>
``` 
This would make the code a bit more concise without affecting functionality.